tiledb/tests/test_libtiledb.py

```diff
@@ -587,25 +587,25 @@ def test_dense_array_schema(self):
         )
         a1 = tiledb.Attr("val", dtype="f8")
         schema = tiledb.ArraySchema(domain=domain, attrs=(a1,))
-        self.assertFalse(schema.sparse)
-        self.assertEqual(schema.cell_order, "row-major")
-        self.assertEqual(schema.tile_order, "row-major")
-        self.assertEqual(schema.domain, domain)
-        self.assertEqual(schema.ndim, 2)
-        self.assertEqual(schema.shape, (8, 8))
-        self.assertEqual(schema.nattr, 1)
-        self.assertEqual(schema.domain.homogeneous, True)
-        self.assertEqual(schema.attr(0), a1)
-        self.assertTrue(schema.has_attr("val"))
-        self.assertFalse(schema.has_attr("nononoattr"))
-        self.assertEqual(schema, tiledb.ArraySchema(domain=domain, attrs=(a1,)))
-        self.assertNotEqual(
-            schema, tiledb.ArraySchema(domain=domain, attrs=(a1,), sparse=True)
-        )
+        assert schema.sparse == False
+        assert schema.cell_order == "row-major"
+        assert schema.tile_order == "row-major"
+        assert schema.domain == domain
+        assert schema.ndim == 2
+        assert schema.shape == (8, 8)
+        assert schema.nattr == 1
+        assert schema.domain.homogeneous == True
+        assert hasattr(schema, "version")  # don't pin to a specific version
```

Comment:
We could look at a fragment (eg `_10`) and check that it matches `.version`, but not strictly necessary to do that.